src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/Context/VedtaksperiodeMedBegrunnelserContext.ts

```diff
@@ -56,8 +67,37 @@ const [VedtaksperiodeMedBegrunnelserProvider, useVedtaksperiodeMedBegrunnelser]
 
         const fritekster = useFelt<FeltState<IFritekstFelt>[]>({
             verdi: [],
-            valideringsfunksjon: (felt: FeltState<FeltState<IFritekstFelt>[]>) =>
-                felt.verdi.length > 1 ? feil(felt, 'Kun 1 fritekst er tillatt') : ok(felt),
+            avhengigheter: { begrunnelser },
+            valideringsfunksjon: (
+                felt: FeltState<FeltState<IFritekstFelt>[]>,
+                avhengigheter?: Avhengigheter
+            ) => {
+                const erEtTomtFritekstfelt = felt.verdi.some(
+                    fritekst => fritekst.verdi.tekst.length === 0
+                );
+                const erEtForLangtFritekstfelt = felt.verdi.some(
```

Comment:
```suggestion
                const harFritekstGyldigLengde = felt.verdi.some(
```
Her kan vi egentlig bare sjekke om noen av fritekstene har !== OK status. Og da sette en generell feilmelding.